xml/System.Windows.Forms/FolderBrowserDialog.xml

```diff
@@ -124,8 +124,8 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets or sets a value that indicates whether the dialog will be automatically upgraded to enable new features.</summary>
+        <value><see langword="true" /> if the the dialog will be automatically upgraded to enable new features; otherwise, <see langword="false" />.</value>
```

Comment:
The default is `true` according to the source:

[src/System.Windows.Forms/src/System/Windows/Forms/FolderBrowserDialog.cs#L46](https://github.com/dotnet/winforms/blob/abd5a3d167ef57ee478912e97f194ff3a21d14b2/src/System.Windows.Forms/src/System/Windows/Forms/FolderBrowserDialog.cs#L46)

```cs
        /// <summary>
        ///  Gets or Sets whether the dialog will be automatically upgraded to enable new features.
        /// </summary>
        [DefaultValue(true)]
        public bool AutoUpgradeEnabled { get; set; } = true;
```

Here's my suggestion:

```suggestion
        <value><see langword="true" /> if the the dialog will be automatically upgraded to enable new features; otherwise, <see langword="false" />. The default value is <see langword="true" />.</value>
```

Reply:
@rpetrusha @mairaw feel free to edit my suggestion if necessary.